### PR TITLE
feat: add transitions and Maverick Tse plugins

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add images for transitions and the Maverick Tse plugins.

## How Has This Been Tested?

The packages can be installed by apm on the main branch.

## Checklist:

- [x] Updated the modification date in `mod.xml`.

### CODE REVIEW

File: v2/data/packages.xml [hunk 1]
```diff
@@ -1717,4 +1909,59 @@
...
+
+	<package>
+		<id>WDee/translib2</id>
+		<name>吉里吉里２トランジションライブラリ</name>
+		<overview>トランジション素材として使用できるPNG画像</overview>
+		<description
+		>トランジション素材として使用できるPNG画像です。シーンチェンジやワイプに対して指定できます。</description>
+		<developer>W.Dee</developer>
+		<pageURL>https://krkrz.github.io/</pageURL>
+		<downloadURL>https://krkrz.github.io/</downloadURL>
```
Comment (hal-shu-sato) [inline v2/data/packages.xml:1923]:
```suggestion
		<downloadURL>https://krkrz.github.io/#krkr2</downloadURL>
```
I found an anchor on the page.
So it is better to use it and assist users.

**Review by hal-shu-sato — COMMENTED** [review]
I have a suggestion.

File: v2/data/packages.xml (hunk 1 above)
Comment (mitosagi) [reply v2/data/packages.xml]: I hadn't noticed. Thank you.

**Review by hal-shu-sato — APPROVED** [review]
Good data.
I didn't come up with the idea to use apm for transition files.
We have to add the type and badge.
Thank you!

**Pull request merged**